Data seeding

leightonchen opened the pull request:
Code review:
Bulk of the work is done in util/seeder.js. Review database schema and the seeding script.

If you want to test:
Create a copy of config.example.js and name it config.js. Put in your mysql user and password, and then run 'npm run seed'. Should populate 100 properties into the properties table and roughly 3500 entries into the photos table (20-50 'photos' per property).

leightonchen committed:
Set up express/webpack

Runs a simple localhost on port 3333, and displays a simple React
component with lorem ipsum text.

leightonchen committed:
Update README

leightonchen committed:
Downloads 100 randomly generated images into the image folder.

Uses axios and loremflickr.com to grab 100 320x240 images with one of
three selected tags. Inserts images into the images folder.

leightonchen committed:
Implement fake data seeding script

Fill out config.js and run 'npm run seed', and the script will
automatically create the tables with proper schema and seed 100 random
properties, with 20-50 photo links each.

bjlagman12 reviewed (approved):
Clean code. Other than some synax I wasn't fully familiar with for sequalize. Everything was straight forward.

SduongPlanD commented:
there seems to be a problem with the seeding code when i tried to run it. here is the error message:

Unhandled rejection SequelizeConnectionError: Unknown database 'xillow'
    at Utils.Promise.tap.then.catch.err (/home/steve/Documents/leighton-component-photo-carousel/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:149:19)

leightonchen committed:
Handle case where no database named xillow exists yet

Opens a connection to create the xillow database if one doesn't exist
yet, before attempting to seed fake data.

SduongPlanD reviewed (approved):
Awesome! looks like a successful seed this time:
3560 new rows in the photos table
100 new rows in the properties table.
nice, looks good.
